pyproject.toml

```
@@ -36,15 +36,23 @@ prometheus_client = "^0.18.0"
 
 # extras
 mlflow-skinny = { version = "^2.0", optional = true }
-redis = {extras = ["hiredis"], version = "^5.0", optional = true}
-boto3 = { version = "^1.24.64", optional = true }
-pydruid= { version= "^0.6", optional = true }
+redis = { extras = ["hiredis"], version = "^5.0", optional = true }
+boto3 = { version = "^1.24.64", optional = false }
+pydruid = { version = "^0.6", optional = true }
+PyMySQL = { version = "^1.1.0", optional = false }
+cryptography = { version = "^42.0.5", optional = true }
+SQLAlchemy = { version = "^2.0.28", optional = true }
+
 
 [tool.poetry.extras]
 mlflow = ["mlflow-skinny"]
 redis = ["redis"]
 dynamodb = ["boto3"]
 druid = ["pydruid"]
+boto3 = ["boto3"]
```

```suggestion
rds = ["boto3", "cryptography", ...]
```
Let's include all together

---

Yes.